client/www/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
I checked the latest tsconfig from create-next-app, and they also set to bundler